### PR TITLE
Fix/images

### PR DESCRIPTION
**Pull request by bobetbat**
- logo and favicon
- loading message

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,11 +2,11 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/images/favicon.ico" />
```
Comment (silksil) [inline public/index.html:5]: I think al favicons are not imported yet in file, for example: 

```  <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/favicon/favicon-32x32.png">


  <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/favicon/favicon-16x16.png">```

File: public/index.html [hunk 2]
```diff
@@ -2,11 +2,11 @@
 <html lang="en">
```
Comment (silksil) [inline public/index.html:2]: maybe an idea to put all the `favicon` images in a seperate `favicon` folder in `public`

Comment (silksil) [reply public/index.html:2]: a suggestion :p

Comment (bobetbat) [reply public/index.html]: removed favicon pngs https://www.javatpoint.com/html-favicon

**Review by kostysh — APPROVED** [review]
LGTM

**Pull request merged**